### ISSUE TITLE
Filter tracks by value

### ISSUE THREAD

**Issue by MerihEkin** [quantumjot/btrack#406]
Would it be possible to have an option to filter tracks (area, time) before running the optimization or drop some tracks from the list

**Comment by quantumjot** [comment]
Absolutely! How do you see this working? Do you want to filter the tracks in Python via your own script?

**Comment by MerihEkin** [comment]
I am using a library for cell detection that includes some noise. For example, I've noticed that the trajectories associated with noise are usually shorter in duration or cover a smaller area. If I could filter out some trajectories—specifically with respect to track length or the convex hull area covered—between the tracking and optimization steps, I believe it would improve the results.

Alternatively, I can easily compute these parameters for each track, so a function to either drop trajectories by specific indices or retain those with specific indices would also be effective.

For instance:

tracker.track(step_size=100)
tracker.filter_tracks_by_index(list_of_track_indices_to_keep/drop)
tracker.optimize()

Thank you for considering this feature and for your efforts in maintaining such a useful library.
